packages/docusaurus-mdx-loader/src/remark/transformImage/index.js

```diff
@@ -35,10 +35,22 @@ const createJSX = (node, pathUrl) => {
   }
 };
 
+/**
+ * Revove replace the starting "../" with "".
+ * E.g: ../package/doc -> package/doc
+ */
+function cleanPath(filePath) {
```

Comment:
## Before adding the `cleanPath` Test Case Result

![image](https://user-images.githubusercontent.com/31756244/95764167-b5089680-0ccd-11eb-9c25-2c91a596d92b.png)

the above test cases are failing because of the extra `../`

![image](https://user-images.githubusercontent.com/31756244/95764464-20eaff00-0cce-11eb-9b9f-9b19c6dbadb3.png)


## After Adding the `cleanPath` Test Case Result

![image](https://user-images.githubusercontent.com/31756244/95764269-d8334600-0ccd-11eb-98e4-a0262c91191e.png)

so, basically `cleanPath`  is just to remove the extra `../` from the path if it exists

Also one more thing all the 4 Test case are passing Completely fine in My Local system even without `cleanPath`.

Reply:
But why are the tests failing due to the extra `../` in the first place? The goal is not only to make the tests pass, also to understand why the solution solves the problem. Where does this extra `../` come from and why only on Windows?

Reply:
> But why are the tests failing due to the extra ../ in the first place? The goal is not only to make the tests pass, also to understand why the solution solves the problem. 

Yeah right, I will look and update.

> Where does this extra ../ come from and why only on Windows?

the extra `../` is not coming in my local System it is here only 😔.

Reply:
Hi @slorber 

I tried finding the reason why  extra `../` is coming.
so it did logging the `process.cwd()` then i found the response different in my local system and different in `Github CI` testing

- **On my local system**

![image](https://user-images.githubusercontent.com/31756244/96153797-b931ef80-0f2b-11eb-8506-d4e688e88f4d.png)


- **On GitHub**

![image](https://user-images.githubusercontent.com/31756244/96153896-d6ff5480-0f2b-11eb-8282-74a78bcb181d.png)


due to this difference in path when we are finding the relative path in our code it is adding the extra `../`

Reply:
oh, this is weird, thanks for letting me know